### PR TITLE
Dashboard + PG IaSQL functions in favor of CLI

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: docs/How-to Guides/export.md [hunk 1]
```diff
@@ -1,12 +0,0 @@
----
```
Comment (depombo) [inline docs/How-to Guides/export.md:1]: https://github.com/iasql/docs/issues/33 to properly revive this

File: docs/ECS Fargate Tutorials/typeorm.md [hunk 2]
```diff
@@ -165,79 +149,38 @@ npx typeorm migration:run
...
+ create | public_repository   |      2 | quickstart-repository
+ create | cluster             |      2 | 2
+ create | task_definition     |      2 | 2
+ create | service             |      2 | 2
+ create | listener            |      2 | 2
+ create | load_balancer       |      2 | 2
+ create | target_group        |      2 | 2
+ create | security_group      |      5 | 5
+ create | security_group_rule |      3 | 3
+ create | security_group_rule |      4 | 4
```
Comment (depombo) [inline docs/ECS Fargate Tutorials/typeorm.md:170]: needs https://github.com/iasql/iasql-engine/issues/575

File: docs/Concepts/module.md [hunk 3]
```diff
@@ -22,4 +22,4 @@ With modules that can conflict and not be installed simultaneously, we actually
...
 
-Modules are managed in IaSQL through the `iasql` CLI. As the [Quickstart](/typeorm) guide indicates, you are prompted when first setting up a hosted IaSQL DB which modules you wish to use, but you can modify that at any time using the `iasql install` or `iasql uninstall` commands. See the [CLI Reference](/cli) for more details.
\ No newline at end of file
+Modules are managed in IaSQL through the PostgreSQL functions loaded into every IaSQL database. As the [Tutorial](/typeorm) guide indicates, you pick which modules you wish to use once you connect to your hosted IaSQL DB. You can cofigure which modules are installed over time using the `SELECT * FROM iasql_install(...)` or ` SELECT * FROM iasql_uninstall(...)` commands. See the [IaSQL PostgreSQL Functions](/function) for more details.
```
Comment (dfellis) [inline docs/Concepts/module.md:25]:
```suggestion
Modules are managed in IaSQL through the PostgreSQL functions loaded into every IaSQL database. As the [Tutorial](/typeorm) guide indicates, you pick which modules you wish to use once you connect to your hosted IaSQL DB. You can configure which modules are installed over time using the `SELECT * FROM iasql_install(...)` or ` SELECT * FROM iasql_uninstall(...)` commands. See the [IaSQL PostgreSQL Functions](/function) for more details.
```

File: docs/ECS Fargate Tutorials/typeorm.md [hunk 4]
```diff
@@ -13,101 +13,85 @@ All the code from this tutorial can be found in this [template repository](https
...
-
-If you ever need to install more modules, you can do so with `iasql install` and to uninstall modules you no longer need, use `iasql uninstall`
-
-:::note
-
-Short command to install these modules if you accidentally didn't select any on DB creation.
-
-```bash
-$ iasql install --db startup aws_cloudwatch aws_ecr aws_ecs aws_elb aws_security_group aws_vpc
+1. Install `psql` in your command line by following the instructions for your corresponding OS [here](https://www.postgresql.org/download/). Remember that many different clients can be used to [connect](/connect) to a PostgreSQL database.
```
Comment (dfellis) [inline docs/ECS Fargate Tutorials/typeorm.md:55]: I think I'd rearrange this like:

```suggestion
1. Many different clients can be used to [connect](/connect) to a PostgreSQL database. For this tutorial, we'll use the standard `psql` CLI client. If you need to install `psql`, follow the instructions for your corresponding OS [here](https://www.postgresql.org/download/).
```

File: docs/Reference/function.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+| -----------| ----------------------------- | --------------------------------------------------------------------- | ------------------------------ |
+| apply      | `iasql_apply()`               | Create, delete or update the cloud resources in a hosted db           | `SELECT iasql_apply()`         |
+| plan_apply | `iasql_plan_apply()`          | Preview of the resources in the db to be modified on the next `apply` | `SELECT iasql_plan_apply()`    |
+| sync       | `iasql_sync()`                | Synchronize the hosted db with the current state of the cloud account | `SELECT iasql_sync()`          |
+| plan_sync  | `iasql_plan_sync()`           | Preview of the resources in the db to be modified on the next `sync`  | `SELECT iasql_plan_sync()`     |
+| install    | `iasql_install(variadic text[])`   | Install mods(s) in hosted db  | `SELECT iasql_install('aws_vpc@0.0.1', 'aws_ec2@0.0.1')`   |
+| uninstall  | `iasql_uninstall(variadic text[])` | Uninstall mod(s) in hosted db | `SELECT iasql_uninstall('aws_vpc@0.0.1', 'aws_ec2@0.0.1')` |
```
Comment (dfellis) [inline docs/Reference/function.md:15]: Nit:

```suggestion
| apply      | `iasql_apply()`               | Create, delete or update the cloud resources in a hosted db           | `SELECT * FROM iasql_apply()`         |
| plan_apply | `iasql_plan_apply()`          | Preview of the resources in the db to be modified on the next `apply` | `SELECT * FROM iasql_plan_apply()`    |
| sync       | `iasql_sync()`                | Synchronize the hosted db with the current state of the cloud account | `SELECT * FROM iasql_sync()`          |
| plan_sync  | `iasql_plan_sync()`           | Preview of the resources in the db to be modified on the next `sync`  | `SELECT * FROM iasql_plan_sync()`     |
| install    | `iasql_install(variadic text[])`   | Install mods(s) in hosted db  | `SELECT * FROM iasql_install('aws_vpc@0.0.1', 'aws_ec2@0.0.1')`   |
| uninstall  | `iasql_uninstall(variadic text[])` | Uninstall mod(s) in hosted db | `SELECT * FROM iasql_uninstall('aws_vpc@0.0.1', 'aws_ec2@0.0.1')` |
```

File: docs/getting-started.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ slug: '/'
...
+[IaSQL](https://iasql.com) is an open-source SaaS to manage cloud infrastructure using a unmodified PostgreSQL database by maintaining a 2-way connection between the hosted database and your AWS account. The rows in the database tables represent the infrastructure in your cloud account. Our [dashboard](https://app.iasql.com) provisions a PostgreSQL database for you and configures it to manage an AWS account and region. The database is automatically backfilled with all your existing cloud resources. Which tables are loaded into an [IaSQL database](/database) is configured based on what [IaSQL modules](/module) are installed in a db. Every IaSQL module represents a cloud service like `aws_ec2` or `aws_elb`. Once the desired modules are installed with the `install` [IaSQL PostgreSQL function](/function), run `INSERT` or `UPDATE` queries on the database by using the PG connection string displayed when `iasql connect` completes with your preferred [PostgreSQL client](/connect). Finally run the `iasql_apply` PostgreSQL function to provision infrastructure in your cloud account based on the IaSQL db.
```
Comment (dfellis) [inline docs/getting-started.md:8]:
```suggestion
[IaSQL](https://iasql.com) is an open-source SaaS to manage cloud infrastructure using a unmodified PostgreSQL database by maintaining a 2-way connection between the hosted database and your AWS account. The rows in the database tables represent the infrastructure in your cloud account. Our [dashboard](https://app.iasql.com) provisions a PostgreSQL database for you and configures it to manage an AWS account and region. The database is automatically backfilled with all your existing cloud resources. Which tables are loaded into an [IaSQL database](/database) is configured based on what [IaSQL modules](/module) are installed in a db. Every IaSQL module represents a cloud service like `aws_ec2` or `aws_elb`. Once the desired modules are installed with the `install` [IaSQL PostgreSQL function](/function), run `INSERT` or `UPDATE` queries on the database by using the PG connection string displayed when you first set up in the dashboard with your preferred [PostgreSQL client](/connect). Finally run the `iasql_apply` PostgreSQL function to provision infrastructure in your cloud account based on the IaSQL db.
```

Comment (dfellis) [reply docs/getting-started.md:8]: (I don't know why Github isn't diffing this line correctly. There was one last `cli` reference in there that I replaced with a reference to the dashboard)

File: docs/ECS Fargate Tutorials/typeorm.md (hunk 4 above)
Comment (depombo) [reply docs/ECS Fargate Tutorials/typeorm.md]: 💯

File: docs/getting-started.md (hunk 6 above)
Comment (depombo) [reply docs/getting-started.md:8]: oof, I see it now. Good catch!

**Pull request merged**